sqlx-macros-core/Cargo.toml

```diff
@@ -65,10 +65,12 @@ serde = { version = "1.0.132", features = ["derive"] }
 serde_json = { version = "1.0.73" }
 sha2 = { version = "0.10.0" }
 syn = { version = "2.0.52", default-features = false, features = ["full", "derive", "parsing", "printing", "clone-impls"] }
-tempfile = { version = "3.10.1" }
 quote = { version = "1.0.26", default-features = false }
 url = { version = "2.2.2" }
 
+[dev-dependencies]
+tempfile = { version = "3.10.1" }
+
```

Comment:
```suggestion
```

@paolobarbolini it's no longer used in `sqlx-macros-core`, you can just delete it outright from here.